src/Hint/NewTypeDerivingStrategies.hs

```diff
@@ -0,0 +1,38 @@
+{-
+    Suggest using explicit deriving strategies for newtype instances
+
+<TEST>
+newtype Foo = Foo Int deriving (Show, Eq) -- newtype Foo = Foo Int deriving newtype (Show, Eq)
+newtype Foo = Foo { getFoo :: Int } deriving (Show, Eq) -- newtype Foo = Foo { getFoo :: Int } deriving newtype (Show, Eq)
+newtype Foo = Foo Int deriving stock Show
+</TEST>
+-}
+module Hint.NewTypeDerivingStrategies (newTypeDerivingStrategiesHint) where
+
+import Hint.Type
+
+newTypeDerivingStrategiesHint :: DeclHint
```

Comment:
I'd just shove this in the Newtype package of hints - it's close enough already and not that long.

Reply:
Done